crate/kmip/src/kmip/kmip_operations.rs

```diff
@@ -1069,6 +1069,11 @@ pub struct ReKeyKeyPair {
     // Storage Mask selections that are permissible for the new Public Key object.
     #[serde(skip_serializing_if = "Option::is_none")]
     pub public_protection_storage_masks: Option<ProtectionStorageMasks>,
+
+    // Specifies the action to perform during the rekeying
```

Comment:
Why not public documentation using `///`? It seems both the structure and the attribute are public.